config/manager/kustomization.yaml

```diff
@@ -3,7 +3,7 @@ resources:
 
 images:
 - name: compliance-operator
-  newName: quay.io/compliance-operator/compliance-operator
+  newName: image-registry.openshift-image-registry.svc:5000/openshift/compliance-operator
```

Comment:
One thing to note about config/manager/kustomization.yaml: The build targets run a kustomize "set image" command that will change this file in your tree, so it should not be committed. (make clean reverts it)

Reply:
done